README.md

```diff
@@ -1,39 +1,34 @@
-radare2-extras
-==============
+# rizin-extras
 
-[![GithubCI Status](https://github.com/radareorg/radare2-extras/workflows/CI/badge.svg?branch=master)](https://github.com/radareorg/radare2-extras/actions?query=workflow%3A%22ci%22)|
+This repository contains extra plugins for Rizin.
 
-This repository contains extra plugins for radare2.
-
-The reasons why those plugins are distributed in a separate
+The reasons why these plugins are distributed in a separate
 repository are the following:
 
 * Depends on external libraries (yara, ewf)
-* Marginal/specific use (ctf-specific asm/anal vm f.ex)
-* r2 must be self-contained (no optional/external deps)
-* Simplify packagers work, and makes it more flexible
-* Duplicates functionality from r2 (no need to have
-  multiple disassemblers for the same arch in core)
-
-The compiled plugins will be installed at
+* Marginal/specific use (ctf-specific assembly/analalysis vm f.ex)
+* Rizin must be self-contained (no optional/external deps)
```

Comment:
Can be removed since it's not really true anymore.

Reply:
All four of them or just the first two?

Reply:
Just the one.